### PR TITLE
Integrate packet-test

### PR DESCRIPTION
**Pull request by cristinaleonr**
This PR updates the client to run a BBR-terminated ndt7 download test unconditionally after the full-length nd7 and MSAK tests are run. The test only takes a couple seconds to complete. The start of the test and the results are logged.

The PR also updates the website language to state that multiple tests are being run.

The data test data is in BigQuery.
```
SELECT * 
FROM `mlab-oti.raw_pt.ndt7`, UNNEST (raw.Download.ClientMetadata) AS metadata 
WHERE date = "2024-08-21" 
AND metadata.Value = "speed-measurementlab-net"
```

The sandbox preview is available [here](https://mlab-sandbox.web.app/).

- - -
This change is [Reviewable](https://reviewable.io/reviews/m-lab/mlab-speedtest/72)

### CODE REVIEW

**Review by stephen-soltesz — APPROVED** [review]
:lgtm:

Please wait for approval from @robertodauria also

> Reviewed 6 of 6 files at r1.
> *[Reviewable](https://reviewable.io/reviews/m-lab/mlab-speedtest/72#-:-O4uhhH-ARXGeVjanZhW:b6z60lu)* status: <span>:shipit:</span> complete! 1 of 1 approvals obtained (waiting on @cristinaleonr and @robertodauria)

___
*[`app/measure/measure.js` line 59 at r1](https://reviewable.io/reviews/m-lab/mlab-speedtest/72#-O4ud7oW1LHX4iSlniPa:-O4ud7oW1LHX4iSlniPb:bw3ewme) ([raw file](https://github.com/m-lab/mlab-speedtest/blob/2adfbb1bd5844264752fb04f4c37460256153dfd/app/measure/measure.js#L59)):*
> ```JavaScript
>       }
> 
>       await runPT(sessionID)
> ```

Just clarifying - this is synchronous from the client's perspective? - if so, and if it runs slowly for any reason, the user would experience a delay before seeing the results from the earlier tests? What's the maximum delay?

**Review by robertodauria — CHANGES_REQUESTED** [review]
Added a comment about randomizing the order of execution. Also, some visual indication that a test is happening would be good -- this is probably more relevant if the order of execution is randomized, since right now the user just sees a delay before the results table appears, but after randomizing the delay might end up at the beginning or between ndt7/msak tests.

> Reviewed 5 of 6 files at r1.
> *[Reviewable](https://reviewable.io/reviews/m-lab/mlab-speedtest/72#-:-O4ui3gt0nSpmwfq8oGc:b-l70ozq)* status: 1 change requests, 0 of 1 approvals obtained (waiting on @cristinaleonr)

___
*[`app/measure/measure.js` line 59 at r1](https://reviewable.io/reviews/m-lab/mlab-speedtest/72#-O4uglOw4XG4gEPsrFBG:-O4uglOw4XG4gEPsrFBH:b-46mrms) ([raw file](https://github.com/m-lab/mlab-speedtest/blob/2adfbb1bd5844264752fb04f4c37460256153dfd/app/measure/measure.js#L59)):*
> ```JavaScript
>       }
> 
>       await runPT(sessionID)
> ```

I believe the order of execution should be randomized in the same way as ndt7/msak. The rationale behind the randomization is that a link where data has been transferred up to half a second ago may exhibit different behavior compared to a link that hasn't seen any significant usage in a while, so any pre-defined order might bias the results.

**Review by cristinaleonr — COMMENTED** [review]
Thank you both. PTAL my latest comment.

> *[Reviewable](https://reviewable.io/reviews/m-lab/mlab-speedtest/72#-:-O4ulu7b73aflXVk2oZD:b-pempje)* status: <span>:shipit:</span> complete! 1 change requests, 1 of 1 approvals obtained (waiting on @robertodauria and @stephen-soltesz)

___
*[`app/measure/measure.js` line 59 at r1](https://reviewable.io/reviews/m-lab/mlab-speedtest/72#-O4ud7oW1LHX4iSlniPa:-O4ul2_F7GwediZAOw1a:b-ky96ka) ([raw file](https://github.com/m-lab/mlab-speedtest/blob/2adfbb1bd5844264752fb04f4c37460256153dfd/app/measure/measure.js#L59)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

Just clarifying - this is synchronous from the client's perspective? - if so, and if it runs slowly for any reason, the user would experience a delay before seeing the results from the earlier tests? What's the maximum delay?
</blockquote></details>

This is a good point. I've made it async so there is no user delay/impact.
___
*[`app/measure/measure.js` line 59 at r1](https://reviewable.io/reviews/m-lab/mlab-speedtest/72#-O4uglOw4XG4gEPsrFBG:-O4ul9xeC-jcnHODRoJh:b-iyum1u) ([raw file](https://github.com/m-lab/mlab-speedtest/blob/2adfbb1bd5844264752fb04f4c37460256153dfd/app/measure/measure.js#L59)):*
<details><summary><i>Previously, robertodauria (Roberto D'Auria) wrote…</i></summary><blockquote>

I believe the order of execution should be randomized in the same way as ndt7/msak. The rationale behind the randomization is that a link where data has been transferred up to half a second ago may exhibit different behavior compared to a link that hasn't seen any significant usage in a while, so any pre-defined order might bias the results.
</blockquote></details>

I agree a randomized order would be ideal. That said, now that it's async it would have to go last.

**Review by robertodauria — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/m-lab/mlab-speedtest/72)* status: <span>:shipit:</span> complete! 1 change requests, 1 of 1 approvals obtained (waiting on @stephen-soltesz)

___
*[`app/measure/measure.js` line 59 at r1](https://reviewable.io/reviews/m-lab/mlab-speedtest/72#-O4uglOw4XG4gEPsrFBG:-O4umxSm9WAEvr0iJI1Q:bm1i6mb) ([raw file](https://github.com/m-lab/mlab-speedtest/blob/2adfbb1bd5844264752fb04f4c37460256153dfd/app/measure/measure.js#L59)):*
<details><summary><i>Previously, cristinaleonr (Cristina Leon) wrote…</i></summary><blockquote>

I agree a randomized order would be ideal. That said, now that it's async it would have to go last.
</blockquote></details>

If scientific accuracy is a concern, I still think this should be randomized, with some visual indication that a test is happening (no need for a percentage or anything, perhaps something like "Running short NDT" since it's just a couple of seconds?) so it can be synchronous and go between existing tests.

I'm not willing to block this PR on that, but noting that this is a very easy criticism to make to any data collected this way.

**Review by robertodauria — APPROVED** [review]

**Review by cristinaleonr — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/m-lab/mlab-speedtest/72)* status: <span>:shipit:</span> complete! 2 of 1 approvals obtained (waiting on @stephen-soltesz)

___
*[`app/measure/measure.js` line 59 at r1](https://reviewable.io/reviews/m-lab/mlab-speedtest/72#-O4uglOw4XG4gEPsrFBG:-O4vG7hY9iOtAG0AwS6C:bujcslc) ([raw file](https://github.com/m-lab/mlab-speedtest/blob/2adfbb1bd5844264752fb04f4c37460256153dfd/app/measure/measure.js#L59)):*
<details><summary><i>Previously, robertodauria (Roberto D'Auria) wrote…</i></summary><blockquote>

If scientific accuracy is a concern, I still think this should be randomized, with some visual indication that a test is happening (no need for a percentage or anything, perhaps something like "Running short NDT" since it's just a couple of seconds?) so it can be synchronous and go between existing tests.

I'm not willing to block this PR on that, but noting that this is a very easy criticism to make to any data collected this way.
</blockquote></details>

Indeed, scientific accuracy is a concern. From the conversations we've been having, my understanding is that user impact is a predominant consideration. Making it go last avoids user disruption and at least makes any bias consistent.

**Pull request merged**
